drivers/frequency/ad9517/ad9517.c

```diff
@@ -153,11 +153,23 @@ int32_t ad9517_setup(struct ad9517_dev **device,
 			return ret;
 	}
 	/* Check if VCO is selected as input. */
-	if(dev->ad9517_st.pdata->vco_clk_sel)
+	if(dev->ad9517_st.pdata->vco_clk_sel) {
 		/* Sets the VCO frequency. */
 		ad9517_vco_frequency(dev,
 				     dev->ad9517_st.pdata->int_vco_freq);
 
+		/* Activate PLL */
+		reg_value =
```

Comment:
for styling consistency, let's do something like this:
```suggestion
		reg_value = AD9517_PLL_POWER_DOWN(0) |
				    AD9517_CP_MODE (3) |
				    AD9517_CP_CURRENT (5) |
				    0 * AD9517_PFD_POLARITY;
```